tests/foreman/cli/test_discoveredhost.py

```diff
@@ -14,50 +14,135 @@
 
 :Upstream: No
 """
-from time import sleep
-
 import pytest
+from wait_for import wait_for
 
-from robottelo.cli.base import CLIReturnCodeError
-from robottelo.cli.discoveredhost import DiscoveredHost
 
 pytestmark = [pytest.mark.run_in_one_thread]
 
 
-def _assertdiscoveredhost(hostname):
-    """Check if host is discovered and information about it can be
-    retrieved back
+@pytest.mark.e2e
+@pytest.mark.on_premises_provisioning
+@pytest.mark.parametrize('module_provisioning_sat', ['discovery'], indirect=True)
+@pytest.mark.parametrize('pxe_loader', ['bios', 'uefi'], indirect=True)
+@pytest.mark.rhel_ver_match('8')
+def test_positive_provision_pxe_host_cli(
```

Comment:
```suggestion
def test_positive_provision_pxe_host(
```
Non-blocking, but we can ignore saying cli since it's in the cli directory.

Reply:
updated